### ISSUE TITLE
Quiz gem not working

### ISSUE THREAD

**Issue by arnoldtonderaimarunda** [jrobertson/quiz#1]
hi @jrobertson 

I'm new to Ruby, and am interested in your quiz gem. 

However, the XML file of the Q&A is being generated okay, but when it is passed to  `rxfhelper`  the following error occurs;
 `/rxfhelper.rb:26:in read: undefined method strip for #<Array:0x007fff090ee4c0> (NoMethodError)`

Trying to bypass `rxfhelper` like your quick example gives the following error;

```
rexml/source.rb:22:in `create_from': Array is not a valid input stream.  It must walk  (RuntimeError)
like either a String, an IO, or a Source.
```

Your assistance is greatly appreciated.

**Comment by jrobertson** [comment]
I fixed the bug, the latest version of the quiz gem is now 0.2.0. The fault was within the RowX gem which could no longer correctly parse the input it was being given in the Test quiz example.  Rather than change the RowX gem, the Quiz gem was updated to parse the input data as well as format it in the strict RowX format which expects a key/value pair for the summary.

**Comment by arnoldtonderaimarunda** [comment]
@jrobertson Thanks for the quick response. I have updated gem to 0.20. However, it not working for me.

I have created the following ruby script

```
require 'quiz'
require 'rexml/document'
require 'rowx'

puts Quiz.new('test.xml').start
```

and created `test.xml` using the xml data you provided in the **readme.md**

However, when I run the script I obtain the following error
_rexml/source.rb:22:in `create_from': Array is not a valid input stream.  It must walk  (RuntimeError)
like either a String, an IO, or a Source._

What am I doing wrong?
